crates/bevy_ui/src/lib.rs

```diff
@@ -39,9 +41,33 @@ pub enum UiSystem {
     Focus,
 }
 
+#[derive(Debug)]
+/// The current scale of the UI for all windows
+///
+/// ## Note
+/// This is purely about the logical scale, and can
+/// be considered like a zoom
```

Comment:
Hmm, I'm not if it's quite a zoom - in my mind it's closer to a shrinking of the logical UI window size. For me, a zoom suggests showing only part of the logical window, which this isn't intended for.

Reply:
As a dev I feel it more intuitive if one considers a zoom, since it effectively multiplies the size of the pixels shown just like a zoom would.

Reply:
This feels better described to me as "setting UI scale", rather than a true zoom.

Reply:
So, remove the note?